src/test/resources/schemas/test.graphqls

```diff
@@ -95,6 +95,12 @@ enum EventStatus {
     LOGGED
 }
 
+# type with directive using enum value
+type User {
+    name: String,
+    friends: [User] @relationship(type: "FRIEND_WITH", direction: OUT)
```

Comment:
Enumeration declaration of `OUT` should be provided, which is very important for Scala. It will have one more import statement. Enumeration not imported is not available. in scala.

Then, class will be:
```
import package.model._
import OUTENUM._
import scala.collection.JavaConverters._

/**
 * type with directive using enum value
 */
@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2020-12-31T23:59:59-0500"
)
case class User(
    name: String,
    @com.example.Relationship(type = "FRIEND_WITH", direction = OUT)
    friends: scala.Seq[User]
) {

}
```